### PR TITLE
Remove all references to "tree" which we never ended up using.

### PR DESCRIPTION
**Pull request by ralphbean**
Tests all pass here.

### CODE REVIEW

**Comment by ralphbean** [comment]
See #12.

File: pdc/apps/module/serializers.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 from pdc.apps.common.models import Arch
 from pdc.apps.common.serializers import StrictSerializerMixin, DynamicFieldsSerializerMixin
 from pdc.apps.common.fields import ChoiceSlugField
-from .models import (Tree, UnreleasedVariant, RuntimeDependency,
-                     BuildDependency)
+from .models import (UnreleasedVariant, RuntimeDependency, BuildDependency)
```
Comment (mprahl) [inline pdc/apps/module/serializers.py]: No longer need parentheses here.

**Comment by mprahl** [comment]
I think `test_tree.json` can be removed.

File: pdc/apps/module/views.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        __Response__:
+
+            # paged lists
+            {
+                "count": int,
+                "next": url,
+                "previous": url,
+                "results": [
+                    {
+                        "variant_id": string,
```
Comment (mprahl) [inline pdc/apps/module/views.py:64]: This is missing a lot of parameters.

File: pdc/apps/module/views.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        """
+        __Method__:
+        GET
+
+        __URL__: $LINK:unreleasedvariant-detail:variant_id$
+
+        __Response__:
+
+            {
+                "variant_id": string,
```
Comment (mprahl) [inline pdc/apps/module/views.py:93]: This is missing a lot of parameters.

File: pdc/apps/module/views.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    def create(self, request, *args, **kwargs):
+        """
+        __Method__:
+        POST
+
+        __URL__: $LINK:unreleasedvariant-list$
+
+        __Data__:
+
+            {
```
Comment (mprahl) [inline pdc/apps/module/views.py:118]: This is missing parameters.

File: pdc/apps/module/views.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                        "variant_uid": string,    # required
+                        "variant_name": string,   # required
+                        "variant_type": string,   # required
+                        "variant_version": string,# version of this particular variant
+                        "variant_release": string,# release of this particular variant
+            }
+
+        __Response__:
+
+            {
```
Comment (mprahl) [inline pdc/apps/module/views.py:135]: This is missing parameters.

File: pdc/apps/module/views.py [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                        "variant_name": string, # required
+                        "variant_type": string, # required
+            }
+
+        __Example__:
+
+            curl -X POST -H "Content-Type: application/json" $URL:unreleasedvariant-list$ \\
+                    -d '{ "variant_id": "core", "variant_uid": "Core", "variant_name": "Minimalistic Core", "variant_type": "module", }'
+            # output
+            {
```
Comment (mprahl) [inline pdc/apps/module/views.py]: This is missing parameters.

**Comment by mprahl** [comment]
Just a few minor comments, but otherwise it looks fine.

**Comment by ralphbean** [comment]
Thanks @mprahl.  All your comments should be addressed.

**Pull request merged**